Makefile.main

```diff
@@ -1,21 +1,26 @@
-# Default shell
-SHELL := /bin/bash
-
 # Package configuration
 PROJECT ?=
 COMMANDS ?=
 DEPENDENCIES ?=
-DOCKERFILES ?= Dockerfile:$(PROJECT)
-PRERUN ?=
-POSTRUN ?=
-RACE_ENABLED ?=
-GOFLAGS ?=
 
-# Docker configuration
-DOCKER_ORG ?=
+# Default shell
+SHELL := /bin/bash
+
+# Dockerfiles to be build, list of pairs file/name, example `Dockerfile:my-image`
+DOCKERFILES ?= Dockerfile:$(PROJECT)
+# Docker registry to push the docker image.
+DOCKER_REGISTRY ?= docker.io
+# Docker organization to be used at the docker image name.
+DOCKER_ORG ?= src-d
+# Username name used to login on the docker registry.
 DOCKER_USERNAME ?=
+# Password used to login on the docker registry.
```

Comment:
Are we writing password in plain files? Seems dangerous 😕

Reply:
It' s an environment variable. This file is not to suppose to be edited.